dm_cli/bin/dm

```diff
@@ -269,6 +270,42 @@ def delete_packages(context, data_source: str, path: str):
         executor.map(thread_function, packages)
 
 
+@cli.group("entity", help="Subcommand for working with entities")
+@click.pass_context
+def entity_cli(context):
+    pass
```

Comment:
not worth combining import package and entity behind one "import" yet?
Should be quite simple, just check if its a filer or folder